### DISCUSSION TITLE
Feature to close applications with middle mouse click on taskbar icon

### DISCUSSION THREAD

**Discussion by kp-gl** [valinet/ExplorerPatcher#1350] [Ideas]
Similar to browser tabs it would be very convenient if somebody could close an app (or in particular case many apps repeatedly) by clicking with the middle mouse button on the apps symbol in taskbar.

EDIT: In the meantime I found out that it can be done with an additional installation of the "7+ Taskbar Tweaker"

**Comment by jorbig** [comment]
+1

**Comment by pyrates999** [comment]
Install 7+ Taskbar Tweaker.  It has that feature.

**Reply by Amrsatrio** [reply]
Speaking about 7+TT this will not work with EP's taskbar. So in the future I will have to add its features directly to the code.

**Reply by pyrates999** [reply]
Oh didn't know that.

**Reply by jondaniels** [reply]
7+TT works for me using EP with Windows10 style taskbar.  However it does seem overkill to have 7+TT installed for this single feature (other features of 7+TT are in EP).

It would be great if middle-click functionality -- specifically closing windows -- was implemented directly in EP.

**Reply by jondaniels** [reply]
Edit: 7+TT no longer works with Windows 11 24H2.

Would be really nice to have directly in EP.

**Reply by pyrates999** [reply]
try windhawk (https://windhawk.net/) and it's mods with the EP implemented windows 10 taskbar.
